classes/PodsAPI.php

```diff
@@ -2978,13 +2978,13 @@ public function save_field( $params, $table_operation = true, $sanitized = false
 				 *      @type null|array  $old_options The old field options (if preexisting).
 				 * }
 				 */
-				do_action( 'pods_api_save_field_table_altered', $definition_mode, $pod, $field['type'], $field, [
+				do_action( 'pods_api_save_field_table_altered', $definition_mode, $pod, $field['type'], $field, array(
 					'simple'         => $simple,
 					'definition'     => $definition,
 					'old_name'       => $old_name,
 					'old_definition' => $old_definition,
 					'old_options'    => $old_options,
-				] );
+				) );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: b1d314b97e9e4f999aa00406cdcf36ad</sub>